src/components/chips/js/chipsController.js

```diff
@@ -167,11 +166,17 @@ MdChipsCtrl.prototype.updateChipContents = function(chipIndex, chipContents){
  * Returns true if a chip is currently being edited. False otherwise.
  * @return {boolean}
  */
-MdChipsCtrl.prototype.isEditingChip = function(){
+MdChipsCtrl.prototype.isEditingChip = function() {
   return !!this.$element[0].getElementsByClassName('_md-chip-editing').length;
 };
 
 
+MdChipsCtrl.prototype.isRemovable = function() {
+  return this.readonly ?
```

Comment:
I had that before, and it looked really unclear. I definitely prefer a ternary operator here.  I think it should be also fine with the ternary :smile: